src/main.cpp

```diff
@@ -1,8 +1,7 @@
 #include "main.h"
 
-int main(int argc, char * argv[]) {
+int main() {
     Controller controller = Controller(WIDTH, HEIGHT, ENERGY);
     controller.init();
     controller.run();
-    return 0;
```

Comment:
again, I think this breaks the program

Reply:
It specifically does not.

for the `int main() {}` function, one _can_ return something, esp. errors or similar, but there is no particular need for it.
The operating system will automatically see the program having run without issue and assign it a 0 - no need for unnecessary things

Reply:
and, tbh, a `return 0` isn't particularly fancy or necessary in this part.
so, since it doesn't break anything I removed it